tbx/people/forms.py

```diff
@@ -1,16 +1,25 @@
-from tbx.people import models
-from wagtail.admin.forms import WagtailAdminPageForm
+from django.core.exceptions import ValidationError
 
+from wagtail.admin.forms import WagtailAdminModelForm
 
-class ContactForm(WagtailAdminPageForm):
-    def clean_default_contact(self):
-        default_contact = self.cleaned_data["default_contact"]
 
-        # If user wants to enable the default contact option
-        if default_contact:
-            # Make sure only one default contact existing
-            models.Contact.objects.filter(default_contact=True).update(
-                default_contact=False
-            )
+class ContactAdminForm(WagtailAdminModelForm):
+    def clean(self):
+        cleaned_data = super().clean()
 
-        return default_contact
+        # These fields are needed in order to avoid potential missing info
+        # when the contact is rendered in the site-wide footer
+        key_fields = ["title", "text", "cta", "name", "role", "image"]
+
+        for field_name in key_fields:
```

Comment:
In the current state of the site the default contact is saved without title, text or cta. However, I think it is fine to make sure we update this on production before launch - maybe one to add to the launch checklist ticket.

Reply:
Awesome, thanks! Added a task in the [launch checklist](https://torchbox.monday.com/boards/1192293412/pulses/1349519856).